src/webgpu/api/operation/command_buffer/image_copy.spec.ts

```diff
@@ -171,10 +176,13 @@ class ImageCopyTest extends GPUTest {
   }
 
   generateData(byteSize: number, start: number = 0, offset: number = 0): Uint8Array {
+    return altDataGenerator.generateView(byteSize, start, offset);
+
     const arr = new Uint8Array(byteSize);
-    for (let i = 0; i < byteSize; ++i) {
+    arr.set(altDataGenerator.generateView(byteSize, start, offset));
+    /*for (let i = 0; i < byteSize; ++i) {
       arr[i + offset] = (i ** 3 + i + start) % 251;
-    }
+    }*/
```

Comment:
dead code and commented out code here. Also I think this function is no longer used at all?

Reply:
Done